CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+  - Major: All custom fields now use a single `errorText` prop to determine error state
```

Comment:
For posterity's sake: here is the PR for the bigmaven integration: https://github.com/mavenlink/mavenlink/pull/20641